base/math.jl

```diff
@@ -165,6 +165,8 @@ julia> deg2rad(90)
 deg2rad(z::AbstractFloat) = z * (oftype(z, pi) / 180)
 rad2deg(z::Real) = rad2deg(float(z))
 deg2rad(z::Real) = deg2rad(float(z))
+rad2deg(z::Number) = (z/pi)*180
+deg2rad(z::Number) = (z*pi)/180
```

Comment:
I think it would be better  to have `(z/180)*pi` to avoid overflow in  special cases here.